### PR TITLE
feat(fw): EOF support

### PR DESCRIPTION
**Pull request by winsvega**
## 🗒️ Description
Eof container structures and eof fixtures

## 🔗 Related Issues
Need to align with this PRS

https://github.com/ethereum/execution-spec-tests/pull/494
https://github.com/ethereum/execution-spec-tests/pull/205

Issues
https://github.com/ethereum/execution-spec-tests/issues/500

## ✅ Checklist
- [x] All: Set appropriate labels for the changes.
- [x] All: Considered squashing commits to improve commit history.
- [x] All: Added an entry to [CHANGELOG.md](/ethereum/execution-spec-tests/blob/main/docs/CHANGELOG.md).
- [x] All: Considered updating the online docs in the [./docs/](/ethereum/execution-spec-tests/blob/main/docs/) directory.
- [x] Tests: All converted JSON/YML tests from [ethereum/tests](/ethereum/tests) have been added to [converted-ethereum-tests.txt](/ethereum/execution-spec-tests/blob/main/converted-ethereum-tests.txt).
- [x] Tests: Included the type and version of evm t8n tool used to locally execute test cases:  e.g., ref with commit hash or geth 1.13.1-stable-3f40e65.
- [x] Tests: Ran `mkdocs serve` locally and verified the auto-generated docs for new tests in the [Test Case Reference](https://ethereum.github.io/execution-spec-tests/main/tests/) are correctly formatted.

### CODE REVIEW

File: tests/prague/eip3540_eof_v1/code_validation_function.py [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+            ),
+        ],
+        validity_error="StackUnderflow",
+    ),
+    Container(
+        name="overflow_code_sections_1",
+        sections=[
+            Section(
+                kind=Kind.CODE,
+                data=(Op.CALLF(i + 1) + Op.RETF),
```
Comment (winsvega) [inline tests/prague/eip3540_eof_v1/code_validation_function.py:205]: Op.RETF  without ()  ?

**Comment by winsvega** [comment]
ok need to setup eof_test function to run throught the external eof verification tool 
and fix test generation in eof format and blockchain test format

**Review by danceratopz — COMMENTED** [review]
Just a comment on moving "constants" to a spec.py module that would get picked up by our docs workflow.

File: tests/prague/eip3540_eof_v1/constants.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+MAX_CODE_INPUTS = 127
+
+MAX_CODE_OUTPUTS = 127
+
+MAX_RJUMPV_COUNT = 0xFF
+
+MAX_BYTECODE_SIZE = 24576
+
+MAX_INITCODE_SIZE = MAX_BYTECODE_SIZE * 2
```
[the remark this answers is not in the document]
Comment (marioevz) [reply tests/prague/eip3540_eof_v1/constants.py]: Changed to `spec.py` :+1:

**Review by marioevz — APPROVED** [review]

**Pull request merged**